docker/scheme_a/config/integration.json

```diff
@@ -0,0 +1,71 @@
+{
+  "INBOUND": {
+    "PORT": 4055,
+    "HOST": "0.0.0.0"
+  },
+  "OUTBOUND": {
+    "PORT": 4056,
+    "HOST": "0.0.0.0"
+  },
+  "WSO2_AUTH": {
+    "staticToken": "0706c62f-c022-3c42-8d14-8b5df128876d",
+    "tokenEndpoint": "",
+    "clientKey": "test-client-key",
+    "clientSecret": "test-client-secret",
+    "refreshSeconds": 3600
+  },
+  "REDIS": {
+    "PORT": 6379,
+    "HOST": "redis",
+    "TIMEOUT": 100
+  },
+  "INSPECT": {
+    "DEPTH": 4,
+    "SHOW_HIDDEN": false,
+    "COLOR": true
+  },
+  "SHARED": {
+    "PEER_ENDPOINT": "inbound-thirdparty-scheme-adapter-b:4057",
+    "ALS_ENDPOINT": "0.0.0.0:4002",
+    "QUOTES_ENDPOINT": "0.0.0.0:3002",
+    "TRANSFERS_ENDPOINT": "0.0.0.0:3000",
+    "BULK_TRANSFERS_ENDPOINT": "",
+    "DFSP_ID": "dfsp_a",
+    "DFSP_BACKEND_URI": "0.0.0.0:9000",
+    "DFSP_BACKEND_HTTP_SCHEME": "http",
+    "JWS_SIGN": false,
+    "JWS_SIGNING_KEY": "./secrets/jwsSigningKey.key",
+    "TLS": {
```

Comment:
you need to fix TLS, take a look here: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/config/integration.json#L38